Dockerfile

```diff
@@ -1,20 +1,62 @@
-FROM golang AS build
+FROM golang:1.20.3-alpine3.17 AS build
 
-RUN mkdir /endlessh
-ADD . /endlessh
-WORKDIR /endlessh
-RUN go mod tidy
-RUN go build -o endlessh .
+# environment settings
+ARG GOOS="linux" \
+    CGO_ENABLED="0"
 
-FROM gcr.io/distroless/base
+# copy files to container
+COPY . "/tmp/endlessh-go/"
 
+# set build workdir
+WORKDIR "/tmp/endlessh-go/"
+
+# build app
+RUN \
+    go mod tidy \
+    && go build -o "endlessh-go"
+
+
+FROM alpine:3.17
```

Comment:
We would better not to use apline in the release.

There are some article on the internet telling why.
e.g. https://martinheinz.dev/blog/92